### ISSUE TITLE
SSD mobilenet object detection false positive predictions

### ISSUE THREAD

**Issue by deepadam** [qfgaohao/pytorch-ssd#129]
Hi, i have been developing an object detection model using pytorch, I have 5.000 images of 1 class, there is actually only 1 class that I want to detect via camera frames.

When I train it 200-300 epoch and the loss stops decreasing, I let it inference with camera, it is very good to detect wanted objects, however It makes false positive predictions with 0.99 confidence on very un-related objects, the objects generally one of the RGB, so they are generally full red or green or blue objects.

I have take photos from this object and augment them with my wanted objects, label them and add to the dataset, however it did not work, same situation continues.

1)The augmented image resolution is 1280p but my camera feed gives 480p, can this be the reason?
2)Some of the slightly blurred images from my dataset can cause this?

Wish to have good days.
